packages/bokeh/test_bokeh.py

```diff
@@ -0,0 +1,14 @@
+def test_bokeh(selenium):
```

Comment:
```suggestion
from pytest_pyodide.decorator import run_in_pyodide

@run_in_pyodide(packages=["bokeh"])
def test_bokeh(selenium):
```
please look at what the other package tests do and imitate them.